src/Microsoft.ML.StandardLearners/Standard/ModelStatistics.cs

```diff
@@ -324,7 +325,7 @@ private List<CoefficientStatistics> GetUnorderedCoefficientStatistics(LinearBina
         /// <summary>
         /// Gets the coefficient statistics as an object.
         /// </summary>
-        public CoefficientStatistics[] GetCoefficientStatistics(LinearBinaryModelParameters parent, RoleMappedSchema schema, int paramCountCap)
+        internal CoefficientStatistics[] GetCoefficientStatistics(LinearBinaryModelParameters parent, RoleMappedSchema schema, int paramCountCap)
```

Comment:
>internal [](start = 8, length = 8)

can this stay public? There is no other way to get the coefficient stats.

Reply:
we should have a workitem to have this method public, if this gets checked in as is, and address that in 0.9 

---
In reply to: [242656194](https://github.com/dotnet/machinelearning/pull/1902#discussion_r242656194) [](ancestors = 242656194)

Reply:
We may need to build a public surface to it, but it can't be this specifically. We'll have to figure something else out.